.gitlab-ci.yml

```diff
@@ -24,9 +24,8 @@ setup_environment:
     # Global default, as long as we have a 9.x.x, environment-modules picks
     # the wrong one as default :(
     DEFAULT_GCC_VERSION: "12.2.0"
-    # Needed for Intel compilers, which may re-use some libraries from a
-    # legacy GCC
-    LEGACY_GCC_VERSION: "9.4.0"
+    # Needed for NVIDIA compilers, which lag behind reasonably modern times
+    LEGACY_GCC_VERSION: "11.3.0"
```

Comment:
Whenever we upgrade GCC, we always check CUDA compatibility and sync with CS to have that updated in advance on BB5. Seems like we missed this compatibility aspect this time :(